### PR TITLE
(refactor): Update ViewerInfo component to use new ProfileImage component

### PR DESCRIPTION
**Pull request by kieckhafer**
Impact: **minor**  
Type: **refactor**

## Component
With the addition of the `ProfileImage` component in #318, we are adding that component to the existing `ViewerInfo` component instead of using the internal profile image code we were previously using.

## Breaking changes
None

## Testing
1. Make sure the existing ViewerInfo component looks the same as it did before the change. In addition, make sure you can add a `profileImage` data inside the `viewer` prop and see the image displayed instead of the users initials.

## Test Changes
- I updated the tests and the test snapshots to include the new ProfileImage component.

### CODE REVIEW

**Comment by machikoyasuda** [comment]
🚀 Preview deployed

Built with commit 0602ac767e8f62204ee0959945e22faf49fa1a12

https://deploy-preview-319--stoic-hodgkin-c0179e.netlify.com

File: package/src/components/ViewerInfo/v1/ViewerInfo.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ Renders a user's first name next to their initials.
```
Comment (nnnnat) [inline package/src/components/ViewerInfo/v1/ViewerInfo.md]: I think this component overview could be updated

**Comment by aldeed** [comment]
@kieckhafer Maybe also move `ViewerInfo` into the new Account section of the sidebar?

**Comment by kieckhafer** [comment]
@nnnnat @aldeed both comments addressed

**Pull request merged**